Packs/AWS-IAM/Integrations/AWS-IAM/AWS-IAM.py

```diff
@@ -679,9 +679,14 @@ def get_role(args, aws_client):  # pragma: no cover
     })
 
     raw = json.loads(json.dumps(response['Role'], cls=DatetimeEncoder))
-    ec = {'AWS.IAM.Roles': raw}
+    raw["Tags"] = raw.get("Tags", [])
     human_readable = tableToMarkdown('AWS IAM Roles', data)
-    return_outputs(human_readable, ec)
+    return CommandResults(
+        outputs=createContext(raw),
+        outputs_prefix='AWS.IAM.Roles',
```

Comment:
is this by design? the dot between aws.iam

Reply:
Yes, this is the same output prefix for all commands in this integration.